### ISSUE TITLE
User for Zoi only with search capability (user limit)

### ISSUE THREAD

**Issue by SoftTools59654** [brimdata/zui#2899]
Hi

Maybe this feature is not very important, but in some cases, users mistakenly delete pools, I think the ability to create a user who can only see and search the data, but **cannot delete** it, would be a good option.

Limited user

**Comment by philrz** [comment]
@SoftTools59654: Indeed, this is a request that has come up a couple times before. Specifically we plan to eventually implement basic [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control) as a feature of the Zed lake, and one level of granularity in the controls would be to separate "read/write" users (that could do things like delete data/pools) from "read-only" users. This would be wired up to the [authentication layer](https://zed.brimdata.io/docs/integrations/zed-lake-auth) that's currently implemented using Auth0 as I mentioned in #2898. Issue https://github.com/brimdata/zed/issues/4892 on the Zed side tracks the work we hope to eventually do there. I'll keep this issue open as a reminder that we'll need to make sure this is supported all the way up into the Zui app experience.
